socorro/processor/signature_utilities.py

```diff
@@ -93,6 +93,63 @@ def __init__(self, config, quit_check_callback=None):
         self.fixup_comma = re.compile(r',(?! )')
 
     #--------------------------------------------------------------------------
+    def _is_exception(
+        self,
+        exception_list,
+        remaining_original_line,
+        line_up_to_current_position
+    ):
```

Comment:
to define that method within ```_collapse``` would imply that it is a closure like the other method ```append_if_not_in_collapse_mode```.  In addition, to define it within ```_collapse``` makes unpleasant depth of indentation in a method that already goes uncomfortably deep.

it could though be defined as a ```staticmethod```, but that doesn't really accomplish much important.